devbuild.sh

```diff
@@ -450,7 +450,7 @@ else
     module use ${SRW_DIR}/modulefiles
     module load ${MODULE_FILE}
     if [[ "${MODULE_FILE}" == "build_macos_gnu" ]]; then
```

Comment:
Suggesting the following for the line 452:
if [[ "${PLATFORM}" == "macos" ]]; then

Reply:
Thanks for your comments. The modifications are committed as you suggested. Please review them again.